src/server/game/Accounts/AccountMgr.cpp

```diff
@@ -21,6 +21,9 @@ namespace AccountMgr
         if (utf8length(username) > MAX_ACCOUNT_STR)
             return AOR_NAME_TOO_LONG;                           // username's too long
 
+        if (utf8length(password) > MAX_PASS_STR)
+            return AccountOpResult::AOR_PASS_TOO_LONG;          // password's too long
+
```

Comment:
How its related to the PR?

Reply:
No it was my bad, i've created the PR with my fork on master branch, now this PR is merged than this snippet will not be considered.